### PR TITLE
Update to use pip, and add noarch

### PR DESCRIPTION
**Pull request by astrofrog**
Replaces #5

### CODE REVIEW

**Comment by bsipocz** [comment]
Nice!

File: recipe/setup.cfg.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git setup.cfg setup.cfg
+index 49917a6..0651f4c 100644
+--- setup.cfg
++++ setup.cfg
+@@ -15,6 +15,8 @@ addopts = -p no:warnings
+ 
+ [ah_bootstrap]
+ auto_use = True
++offline = True
```
Comment (bsipocz) [inline recipe/setup.cfg.patch:9]: maybe add a reminder comment that this patch can be removed once https://github.com/pypa/pip/issues/4383 is fixed?

**Comment by astrofrog** [comment]
@conda-forge-admin please rerender

**Comment by astrofrog** [comment]
@conda-forge-admin please rerender

File: recipe/setup.cfg.patch (hunk 1 above)
Comment (astrofrog) [reply recipe/setup.cfg.patch:9]: Done

**Comment by bsipocz** [comment]
All green 🎉, thanks @astrofrog!

**Pull request merged**